Automattic/newspack-newsletters
# Add plugin readme.txt

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds a readme.txt file to the plugin, following the format from these sources:

https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/
https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/

Right now, the 'Changelog' part is copy-pasted from the release changelogs, down to v1.0 (which I replaced with 'Initial release', to save having to copy over all of the items). In the future it could be handy to automate this somehow, but this should work for the short-term. 

Note that the different sections ("Description", "Installation", "Installation") populate the different tabs in the plugin repo, like on the Jetpack page: https://wordpress.org/plugins/jetpack/

### How to test the changes in this Pull Request:

1. Read through the readme and check for any errors. 
2. Check for any formatting issues in the markdown eg. make sure the links are formatted correctly. 
3. Confirm if the required PHP, WordPress versions are correct. 

Open-ended questions: is anything missing? 

### Other information:
